[2022-06-27 07:24:02,461] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: test.training_model_C scheduled__2022-06-26T00:00:00+00:00 [queued]>
[2022-06-27 07:24:02,470] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: test.training_model_C scheduled__2022-06-26T00:00:00+00:00 [queued]>
[2022-06-27 07:24:02,470] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 07:24:02,470] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-06-27 07:24:02,470] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 07:24:02,479] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): training_model_C> on 2022-06-26 00:00:00+00:00
[2022-06-27 07:24:02,485] {standard_task_runner.py:52} INFO - Started process 1928 to run task
[2022-06-27 07:24:02,487] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'test', 'training_model_C', 'scheduled__2022-06-26T00:00:00+00:00', '--job-id', '89', '--raw', '--subdir', 'DAGS_FOLDER/second_2.py', '--cfg-path', '/tmp/tmpx7ty4qfw', '--error-file', '/tmp/tmpzudlar1q']
[2022-06-27 07:24:02,488] {standard_task_runner.py:80} INFO - Job 89: Subtask training_model_C
[2022-06-27 07:24:02,526] {logging_mixin.py:109} INFO - Running <TaskInstance: test.training_model_C scheduled__2022-06-26T00:00:00+00:00 [running]> on host f32f6d4577d5
[2022-06-27 07:24:02,566] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test
AIRFLOW_CTX_TASK_ID=training_model_C
AIRFLOW_CTX_EXECUTION_DATE=2022-06-26T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-26T00:00:00+00:00
[2022-06-27 07:24:02,567] {python.py:175} INFO - Done. Returned value was: 10
[2022-06-27 07:24:02,585] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=test, task_id=training_model_C, execution_date=20220626T000000, start_date=20220627T072402, end_date=20220627T072402
[2022-06-27 07:24:02,619] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-06-27 07:24:02,651] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
